File: app/src/test/java/sqa_project/GeminiPro/decorator/round1/NavigationSystem.java
package sqa_project.GeminiPro.decorator.round1;

public class NavigationSystem extends CarDecorator {
    public NavigationSystem(Car decoratedCar) {
        super(decoratedCar);
    }

    @Override
    public String getDescription() {
        return super.getDescription() + ", Navigation System";
    }

    @Override
    public double getCost() {
        return super.getCost() + 1000;
    }
}

File: Python/ChatGPT4o/Decorator/round1/NavigationSystem.py
from CarDecorator import CarDecorator

class NavigationSystem(CarDecorator):
    """Decorator class for adding a navigation system to a car."""

    def description(self):
        """Adds a navigation system to the car description."""
        return self._car.description() + ", Navigation System"

    def cost(self):
        """Adds the cost of a navigation system to the car."""
        return self._car.cost() + 2500